### PR TITLE
[SS-367]: Publish a GTM Template for the chat

### PR DESCRIPTION
**Pull request by MaGOs92**
### Linear issue : <a href="https://linear.app/gorgias/issue/SS-367/publish-a-gtm-template-for-gorgias-chat">SS-367</a>

Docs to publish a GTM Template : https://developers.google.com/tag-manager/templates/gallery

### Changes
- Add a GTM template file
- Add a metadata.yaml file (to update once template is ready)
- Add an Apache 2 license

### Tests (With staging)
- The tag used for this test has been created from the GTM Template in https://tagmanager.google.com/?authuser=1#/container/accounts/6003048376/containers/39256078/workspaces/19
- Make sure that your ad block is turned off
- Browse to https://gfay-gtm-test.myshopify.com/ (password: awnewd)
- The chat should be loaded via GTM

### TODO
- Update metadata.yaml with the release commit once ready
- Create a documentation on https://docs.gorgias.com/ to use the template

### CODE REVIEW

File: gorgias-chat-template.tpl [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+{
+  "type": "TAG",
+  "id": "cvt_temp_public_id",
+  "version": 1,
+  "securityGroups": [],
+  "displayName": "Gorgias Chat",
+  "categories": ["CHAT", "SALES"],
+  "brand": {
+    "id": "brand_dummy",
```
Comment (vicdup) [inline gorgias-chat-template.tpl]: do we want to keep dummy?

**Review by vicdup — CHANGES_REQUESTED** [review]

File: gorgias-chat-template.tpl (hunk 1 above)
Comment (MaGOs92) [reply gorgias-chat-template.tpl]: I guess no^^ I changed it to "gorgias"

**Comment by vicdup** [comment]
Code LGTM, what's next? Let's do it!

**Review by vicdup — APPROVED** [review]

**Comment by MaGOs92** [comment]
The next step is to create a documentation for the template in the docs. Because we need to provide a link to a documentation page in the file metadata.yaml (and specify the commit to release). I think updating this page (https://docs.gorgias.com/gorgias-chat/chat-getting-started) with GTM instructions would be the best.

For the moment we could have something like this : 

### Install Gorgias Chat using Google Tag Manager
1. Import the Gorgias Chat template in your GTM workspace. You can do this either from the <a href="https://tagmanager.google.com/gallery">GTM gallery</a> page or directly from your GTM workspace when creating a new tag and selecting "Discover more tag types in the Community Template Gallery".
2. Create a Tag using Gorgias Chat Template. To do so, you need to provide your Gorgias Chat ID. You can get this information in the HelpDesk at this path : Integrations > Chat > Your Chat > Installation > Custom installation tab Google Tag Manager.
3. Submit the modifications.

Then once the template is released, we could add screenshots to the documentation.

And then I think we can submit the template. To do so we can submit this repo here : https://tagmanager.google.com/ => Menu => Submit a template

**Comment by MaGOs92** [comment]
I opened a PR https://github.com/gorgias/gorgias/pull/9276 for the update on the help desk (using basically the same instructions)

**Pull request merged**